src/lib.rs

```diff
@@ -63,3 +242,16 @@ impl Bridge {
         self.auth0_opt.as_ref().map(|auth0| auth0.token())
     }
 }
+
+mod sealed {
+    use crate::BridgeClient;
+
+    pub trait Sealed {}
```

Comment:
What about [this](https://crates.io/crates/sealed)?

Reply:
Just for the sake of being supersafe, have you checked that from outside you can used the types that implement the `Sealed` trait. I guess you can but a try could be worth it..

Reply:
Do the existing tests cover the usage you are thinking? They're unable to implement `Sealed` but make use of sealed types, I think?

Reply:
Yes, you are right. Since tests under `tests` folder are able to access only the high level api i will say it's ok :)